### ISSUE TITLE
Make nvda advertise end when navigating character by character during screen exploration

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9351]
to reproduce:
When using the screen exploration and browsing line by line with nvda + arrow assume and below and navigating by words with nvda + ctrl + right and left arrows.
The nvda when it finds the end informs end.
But when navigating with nvda + right and left arrow the nvda does not announce that it came at the end or at the beginning.
### Actual behavior:
When navigating with nvda  + right or left arrows in the screen exploration mode, nvda repeats the character at the beginning and at the end plus does not tell start and end.
### Expected behavior:
when navigating with nvda + right or left arrows and finding the start or end the nvda should announce start or end.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2018.4.1
#### Windows version:
10 17134
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
yes

**Comment by Adriani90** [comment]
@fernando-jose-silva I cannot reproduce the issue. When I use nvda+left and right arrow, NVDA tells me when I am at the left or right side of the line. Did you try it with NVDA language set on inglish? Did you try another synthesizer? Which addons do you have installed?

**Comment by fernando-jose-silva** [comment]
I'm on another machine with the Windows 10 version 17763 different and this worked.
If you can wait until Monday please, I will have contact with the machine with problems for new

**Comment by fernando-jose-silva** [comment]
Thank you all.
I solved the problem, there was a problem with the installation of nvda, I reinstalled nvda and everything works as it should.

**Comment by Adriani90** [comment]
Thanks for reporting back. Closing as works for me.
